front/pages/api/w/[wId]/assistant/conversations/[cId]/content_fragment/index.ts

```diff
@@ -0,0 +1,165 @@
+import { isLeft } from "fp-ts/lib/Either";
+import * as t from "io-ts";
+import * as reporter from "io-ts-reporters";
+import { NextApiRequest, NextApiResponse } from "next";
+
+import {
+  getConversation,
+  postNewContentFragment,
+} from "@app/lib/api/assistant/conversation";
+import { Authenticator, getSession } from "@app/lib/auth";
+import { ReturnedAPIErrorType } from "@app/lib/error";
+import { apiError, withLogging } from "@app/logger/withlogging";
+import { ContentFragmentType } from "@app/types/assistant/conversation";
+
+export const PostContentFragmentRequestBodySchema = t.type({
+  title: t.string,
+  content: t.string,
+  url: t.union([t.string, t.null]),
+  contentType: t.union([
+    t.literal("slack_thread_content"),
+    t.literal("file_attachment"),
+  ]),
+  context: t.type({
+    timezone: t.string,
```

Comment:
why take timezone ?
why take profilePictureUrl only ?

From slack we want 0 context. It's not user related.

For user provided contentFragment we want user information which I think should cover as much as possible (userame display name so that we can render a hover if needed).

Timezone is not needed here so I would overall push here profile picture username and name but not timeZone. Not push anything from slack. And in DB remove timeZone

Reply:
Cleaned up

Reply:
It seems there's still timezone & not other fields? maybe I missed something you discussed irl?

Reply:
Removed. thanks.